### ISSUE TITLE
Suggestions for the graph layout

### ISSUE THREAD

**Issue by BurnzZ** [scrapinghub/varanus#1]
In the image below, we typically have this type of graph:
![image](https://user-images.githubusercontent.com/3449761/65243935-09b5fe00-db1c-11e9-90b4-43efc040be70.png)

In order for the graphs to be more easily understandable, here are two (2) suggestions:
1. Make the graphs **left-to-right** by default where the newer ones are on the right. Another way to look at this would be that the job keys would be in ascending order.
2. Instead of a vertically-stacked graph, could there be perhaps an option to have a horizontally-stacked like the one below? This will make eye-ball comparing a lot easier since 
 the bars all start with with the same base zero. In other words, it's very easy to see which bars is higher than the other in this layout.
![image](https://user-images.githubusercontent.com/3449761/65244273-ae384000-db1c-11e9-9bd4-d7cc2f8f1220.png)

**Comment by stav** [comment]
@BurnzZ I agree.  The graph displays any numbers in the data result of the given command which is seen by removing `-f graph` flag. Have you tried limiting the columns displayed by using the `-c` flag? For example: `-c items`.
